torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -865,7 +865,16 @@ void TensorExprKernel::LowerToBackend(BackendType backend_type) {
       Tensor* tensor = tensor_outputs_[i];
       ExprHandle total_count = ExprHandle(tensor->dim(0));
```

Comment:
I wonder whether we could have a 0-dim tensor here.  Feels like something I should have hit in testing.  Probably not super relevant for this diff.